Makefile

```diff
@@ -92,6 +92,7 @@ sam-deploy-package: guard-artifact_bucket guard-artifact_bucket_prefix guard-sta
 			  LogLevel=$$LOG_LEVEL \
 			  LogRetentionInDays=$$LOG_RETENTION_DAYS \
 			  Env=$$TARGET_ENVIRONMENT \
+			  Environment=$$TARGET_ENVIRONMENT \
```

Comment:
duplicate variable with the same value? Is this intentional?

Reply:
It wasn't accessible from within the sam template so I assumed `Env` was a sam config. I can't see where it's referenced so I've removed it and I'll try deploying to see if it's actually needed

Reply:
Deployment seems ok. I presume it's been copied and pasted from elsewhere and was no longer necessary 🤔